addons/mail/wizard/template_reset_wizard_views.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="template_reset_wizard_view_form" model="ir.ui.view">
+        <field name="name">template.reset.wizard.view.form</field>
+        <field name="model">template.reset.wizard</field>
+        <field name="priority">1000</field>
+        <field name="arch" type="xml">
+            <form>
+                <div>
+                    Are you sure you want to reset these email templates to their original configuration? Changes and translations will be lost.
+                </div>
+                <footer>
+                    <button string="Proceed" class="btn btn-primary" type="object" name="reset_template" data-hotkey="q"/>
+                    <button string="Cancel" class="btn-secondary" special="cancel" data-hotkey="z" />
+                </footer>
+            </form>
+        </field>
+    </record>
+
+    <record id="action_reset_mail_template" model="ir.actions.act_window">
```

Comment:
According to guidelines, should be template_reset_wizard_action (mail_template_reset_action if we change the model name, obviously)